README.md

```diff
@@ -3,129 +3,87 @@
 Cloud KMS (Key Management System) is a hardware-software combined system that provides customers
 with capabilities to create and manage cryptographic keys and control their use for cloud services.
 
-Typically, it is backed with HSM (Hardware Security Module). However,the existing hardware HSM solution
+Typically, the KMS (Key Manangement Service) is backed with HSM (Hardware Security Module). However,the existing hardware HSM solution
 is very expensive and complex to manage. So, many CSPs will provide Cloud HSM solutions to alleviate the
 problem of the scalability and availability issues, but they are still depend on the dedicated external
 hardware devices, and currently there's no opensource with low cost and scalable secure HSM solution for
 Cloud KSM scenario.
 
-This project is called eHSM, which is provided as a reference code based on Intel Software Guard Extensions(SGX)
-technology. It's designed to be protected from processes running at higher privilege levels, like OS kernel,
-hypervisor, BIOS firmware.
+This project is called eHSM-KMS that provides as a reference codes for helping the user to setup a customized KMS (Key Management System) which has the capabilities to create and manage cryptographic keys for the customer. And this solution is implemented based on the Intel SGX (Intel Software Guard Extensions), which could provide the equivalent protection level of cryptographic functionalities, including key generation, management, use etc. inside the SGX enclave. 
```

Comment:
This project is named eHSM-KMS, which provides a reference to help the users setup a customized KMS
 (Key Management System) with the capabilities to create and manage cryptographic keys. 
 It's based on Intel SGX (Software Guard eXtensions) technology to provide the equivalent protection level of cryptographic functionalities 
 including key generation, management or using inside the SGX enclave.